README.md

```diff
@@ -327,6 +327,32 @@ To install the completions for fish, run
 sudo cp alacritty-completions.fish $__fish_datadir/vendor_completions.d/alacritty.fish
 ```
 
+## Terminfo
+
+The terminfo database contains entries describing the terminal
+emulator's capabilities. Programs need these in order to function
+properly.
+
+Alacritty should work with the standard `xterm-256color` definition,
+but to allow programs to make best use of alacritty's capabilities,
+use it's own terminfo definition instead.
+
+Note that the default configuration **requires** alacritty's terminfo.
+
+To install alacritty's terminfo entry globally:
+
+```sh
+sudo tic alacritty.info
+```
+
+If you are a package maintainer, you might want to do something along
+these lines instead (`${pkgdir}` refers to the directory where
+the package is being built):
+
+```sh
+tic -o "${pkgdir}/usr/share/terminfo" alacritty.info
+```
```

Comment:
I don't think it's necessary to document what maintainers are supposed to do here. There should be plenty of examples and other documentation with the package manager they're using. And they might follow completely different conventions anyways.

So I think it's safe to drop this part to keep the readme a bit more minimal.